res/sectors.xsd

```diff
@@ -145,13 +148,39 @@
                   <xs:complexType>
                     <xs:simpleContent>
                       <xs:extension base="xs:string">
-                        <xs:attribute name="Allegiance" type="xs:string" use="required" />
+                        <xs:attribute name="Allegiance" type="xs:string" use="optional" />
                         <xs:attribute name="Color" type="xs:string" use="optional" />
                         <xs:attribute name="Style" type="xs:string" use="optional" />
                         <xs:attribute name="Label" type="xs:string" use="optional" />
                         <xs:attribute name="ShowLabel" type="xs:boolean" use="optional" />
                         <xs:attribute name="WrapLabel" type="xs:boolean" use="optional" />
                         <xs:attribute name="LabelPosition" type="xs:unsignedShort" use="optional" />
+                        <xs:attribute name="LabelOffsetX" type="xs:float" use="optional" />
+                        <xs:attribute name="LabelOffsetY" type="xs:float" use="optional" />
+                        <xs:attributeGroup ref="metadata"/>
+                      </xs:extension>
+                    </xs:simpleContent>
+                  </xs:complexType>
+                </xs:element>
+              </xs:sequence>
+              <xs:attributeGroup ref="metadata"/>
+            </xs:complexType>
+          </xs:element>
+          <xs:element name="Regions">
+            <xs:complexType>
+              <xs:sequence>
+                <xs:element minOccurs="0" maxOccurs="unbounded" name="Region">
+                  <xs:complexType>
+                    <xs:simpleContent>
+                      <xs:extension base="xs:string">
+                        <xs:attribute name="Allegiance" type="xs:string" use="optional" />
```

Comment:
I'm not sure if this should be present or not. In metadata.html it says it's not typically used which would suggest it's still valid.

Related to this, metadata.html says Style is ignored for Regions so I've not added to the xsd.